src/frontend/src/components/message.ts

```diff
@@ -0,0 +1,33 @@
+import { html, TemplateResult } from "lit-html";
+import { ifDefined } from "lit-html/directives/if-defined.js";
+import { renderPage } from "../utils/lit-html";
+
+const showMessageTemplate = ({
+  message,
+  role,
+}: {
+  message: string;
+  role?: string;
+}): TemplateResult => {
+  return html`<h1
+    style="position: absolute; max-width: 100%; top: 50%; transform: translate(0, -50%);"
```

Comment:
not for this PR: shouldn't it be a `class`?

Reply:
yeah I think that'd make sense. I was just moving code around though but happy to do this in this PR if you prefer